task_DEV-3/EntryPoint.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace task_DEV_3
+{
+    /// <summary>
+    /// An entry point in the program.
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                Inputter inputter = new Inputter();
+                inputter.InputForConverter(args);
+
+                Converter converter = new Converter();
+
+                Console.WriteLine(inputter.GetNumber() + "  " + inputter.GetNewBase());
+                Console.WriteLine(converter.Convert(inputter.GetNumber(), inputter.GetNewBase()));
+            }
+            catch (Exception e)
+            {
```

Comment:
Why catch and then throw? in this case exception will be processed by operation system